grails-app/controllers/org/pih/warehouse/api/StockTransferApiController.groovy

```diff
@@ -67,7 +67,7 @@ class StockTransferApiController {
         bindStockTransferData(stockTransfer, currentUser, currentLocation, jsonObject)
 
         Order order = stockTransferService.createOrUpdateOrderFromStockTransfer(stockTransfer)
-        if (order.hasErrors() || !order.save(flush: true)) {
```

Comment:
Why this save was removed?

Reply:
This save throws `No transaction in progress error` which I wasn't sure why.
I looked inside `stockTransferService.createOrUpdateOrderFromStockTransfer()` method and found that `order.save()` is already being triggered there so I assumed that this might be the problem.